pom.xml

```diff
@@ -18,7 +18,7 @@
   <groupId>co.cask.hydrator</groupId>
   <artifactId>loadtosnowflake-action-plugin</artifactId>
   <modelVersion>4.0.0</modelVersion>
-  <version>1.0.0-SNAPSHOT</version>
+  <version>1.0.1</version>
```

Comment:
Should we add `SNAPSHOT` version on develop?

Reply:
Normally yes, bit the next PR will immediately bump it to 1.1.0, so its pointless in this case